bigtop_toolchain/manifests/packages.pp

```diff
@@ -204,7 +204,6 @@
         "bison",
         "flex",
         "python-dev",
-        "python-pip",
```

Comment:
Can we remove pip (for Python 2) from all distributions?

Reply:
Yes I think so. Python2 is still used in some components, but pip2 is not as far as I can see.

Reply:
I think we should see the effect on Ubuntu/Debian first by committing this as is.